### DISCUSSION TITLE
engine future side effect when False

### DISCUSSION THREAD

**Discussion by priamai** [fastapi/sqlmodel#1460] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
engine = create_engine(settings.DATABASE_URI, echo=True, connect_args=connect_args,future=False)

data_df = pd.read_sql(table_name, con=engine, parse_dates=['datetime'])
```

### Description

I am using FASTAPI and SQLModel with Pandas.
I create dataframes and convert them to json before returning them.
I need to set the future = False otherwise pandas is unable to read sql.
Will this have any side effects on the framework?

### Operating System

Linux

### SQLModel Version

0.0.8

### Python Version

3.8.10

**Comment by faizanazim11** [comment]
Can you share the library versions that you are using?

i.e:

* Pandas
* SQLAlchemy
* Psycopg or any other DBAPI
